recipes/harfbuzz/all/conanfile.py

```diff
@@ -77,11 +80,11 @@ def validate(self):
             raise ConanInvalidConfiguration(
                 "Linking a shared library against static glib can cause unexpected behaviour."
             )
-        if scm.Version(self.version) >= "4.4.0":
-            if self.settings.compiler == "gcc" and scm.Version(self.settings.compiler.version) < "7":
+        if Version(self.version) >= "4.4.0":
+            if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "7":
```

Comment:
Is it possible to use the `self.settings.info` and `self.options.info` forms in the validate method here?